trytond/trytond/backend/sqlite/database.py

```diff
@@ -261,16 +263,27 @@ def sign(value):
         return value
 
 
+def prepare_comparison(items):
+    res = []
```

Comment:
Un cht'it commentaire pour expliquer pourquoi est-ce qu'il y a besoin de faire ça ?

Reply:
done, le cas est: 

![float_and_int](https://github.com/coopengo/tryton/assets/16104512/f47273bf-d6ed-4f8a-a1f9-1d9e44e92b4f)

je suis pas hyper convaincu, mais je ne vois pas d'alternative (et pas certain qu'il soit nécessaire d'être plus général)

Reply:
c'est un table_query avec Greatest(line.debit, 0)  dedans. je suis pas hyper convaincu de mon fix